### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by peterrobert**
1. Added test.rb file inside specs folder
2. Tested valid_movement? method in the Game class
3. Tested player_win? method in the Game class
4. Tested finished? method in the Game class
5. Tested new_movement  method in the Game class

It is not necessary to test the board class since it is only has a printing method.
It is not necessary to test the reset method because it only set a new value to attributes in the game class

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  # compatibility in RSpec 3). It causes shared context metadata to be
+  # inherited by the metadata hash of host groups and examples, rather than
+  # triggering implicit auto-inclusion in groups with matching metadata.
+  config.shared_context_metadata_behavior = :apply_to_host_groups
+
+# The settings below are suggested to provide a good initial experience
+# with RSpec, but feel free to customize to your heart's content.
+=begin
+  # This allows you to limit a spec run to individual examples or groups
+  # you care about by tagging them with `:focus` metadata. When nothing
```
Comment (suretrust) [inline spec/spec_helper.rb]: Let's fix the stickler error.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, 👋 

### Great job thus far. 🙌 Please see my suggestions. 👇 

- [x] Ensure that all your tests pass.
Screenshot 2020-03-30 at 18 58 35

- [x] Add `How to run test` to your `readME` file.

- [x] Ensure to test for both negative and positive scenarios. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)). Example:
```ruby
it 'returns true if it recieves a number between 1 and 9 and this number must be unique in first player and second player movements' do
      expect(game.valid_movement?([], [], 9)).to eql(true)
      expect(game.valid_movement?([], [], 9)).not_to eql(false)
end
```

**Kindly submit for another code review request once these issues have been fixed.** _Happy coding!_ 🎉

**Comment by peterrobert** [comment]
Updated the readme file with testing instructions.
Updated the testing methods with negative scenarios.
Fixed the testing errors.

**Comment by Anna-Myzukina** [comment]
## Hi team!  
### Great job! :+1: You can now merge and continue.

## Status: Approved!



#### Goodluck in your next project Happy coding <g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png">computer</g-emoji>

**Review by Anna-Myzukina — APPROVED** [review]

**Pull request merged**